doc/level0.rst

```diff
@@ -161,13 +170,13 @@ kept in short term memory. For instance::
     AB1D-E2GH-IJK3-4NOP-Q5ST-XYZ6
 
 An Autocrypt Setup Message payload that uses this structure for its
-setup code SHOULD include the following header in the outer OpenPGP
-armor::
+setup code SHOULD include the following ``input-type`` attribute in
+the top-level ``Autocrypt-Setup-Message`` header::
```

Comment:
why isn't this in the ascii armored header? it's not autocrypt-specifc, and also something the pgp implementation will have to deal with directly.